### DISCUSSION TITLE
AttributeError: module 'towhee' has no attribute 'glob'

### DISCUSSION THREAD

**Discussion by wangtian256** [towhee-io/towhee#2684] [Q&A]
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
Cell In[27], line 1
----> 1 towhee.glob['path']('./test/lion/n02129165_13728.JPEG').image_decode['path', 'img']().image_embedding.timm['img', 'vec'](model_name='resnet50').select['img', 'vec']().show()

AttributeError: module 'towhee' has no attribute 'glob'
my towhee version is 1.3.1, and Python version is 3.10.9.

**Comment by junjiejiangjjj** [comment]
https://github.com/towhee-io/towhee/issues/2614
